scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -32,6 +32,25 @@
       </tbody>
     </table>
   {% endif %}
+
+  {% if variant.clnsig_onc %}
+    <table id='clinsig_table' class="table table-bordered table-sm">
+      <thead>
+        <th>Oncogenicity</th>
+        <th>Disease name</th>
```

Comment:
Do we need to repeat the ClinVar accession here? For me it is implicit that its the same numbers as in the clnsig table but perhaps it's not obvious?

Reply:
Not really, if they all are the same, but it wasn't too long ago that they merged their individual variant entries to "allele" entries right? Maybe we even have some old cases where they do differ - and I suppose we may not be guaranteed there is a one-to-one correspondence? 😊

Reply:
> Not really, if they all are the same, but it wasn't too long ago that they merged their individual variant entries to "allele" entries right? Maybe we even have some old cases where they do differ - and I suppose we may not be guaranteed there is a one-to-one correspondence? 😊

I don't think we will find ONC fields present on on classification entries. It's new stuff created after the restructuring